polyply/tests/test_gen_params.py

```diff
@@ -124,3 +126,36 @@ def test_find_missing_links():
         assert edge["resB"] == "P3HTref"
         assert edge["idxA"] == ref[0]
         assert edge["idxB"] == ref[1]
+
+@pytest.mark.parametrize('warn_type', ['INFO', 'WARNING', 'ERROR'])
+def test_print_log_warnings(tmp_path, monkeypatch, caplog, warn_type):
+    """
+    Quick test to make sure that the logging warnings propagate to the
+    gen_params output.
+    """
+    # change to temporary direcotry
+    monkeypatch.chdir(tmp_path)
+
+    # get input file from test data
+    infile = TEST_DATA / Path(f"gen_params/logging/{warn_type}.ff")
+
+    # set loglevel
+    loglevel = getattr(logging, warn_type)
+    LOGGER.setLevel(loglevel)
```

Comment:
I'm not sure it makes sense to set the loglevel. What's your reasoning for not setting it to 1 to show *everything*?

Reply:
I tried a few things yesterday; in the end I settled on setting the log level explicitly, because it also checks that setting the log level within polyply works. It's sort of a double check given some of the problems we had in propagating the log level correctly. I think it's fine to be a bit more explicit.